projects/samples/devices/worlds/track.wbt

```diff
@@ -305,7 +306,8 @@ DEF CONVEYOR_BELT Robot {
         LinearMotor {
         }
       ]
-      textureAnimation 0.0295 0
+      # Syntax to write formula in webots files ?
```

Comment:
This is not possible. It's possible only in PROTO files.

Reply:
That's what I thought, thanks for confirmation

Reply:
Don't hesitate to ask such questions on Discord or on the github discussion thread of your PR.